cmd/integration/integration.go

```diff
@@ -114,7 +114,7 @@ func (h *delegateHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 
 func startComponents(manifestURL string) (apiServerURL string) {
 	// Setup
-	servers := []string{"http://localhost:4001"}
+	servers := []string{}
```

Comment:
This was an artifact of the bad behavior of `-addr` (binding on the first address, which localhost can resolve to).  It no longer works now that we are using -bind-addr.